cpp/Ice/context/Client.cpp

```diff
@@ -11,19 +11,23 @@ int
 main(int argc, char* argv[])
 {
     // Figure out my name.
+#ifdef _MSC_VER
```

Comment:
I find this updated getenv gets in the way of the sample. I would rather keep the previous code and disable the warning, or use a fixed name.

Reply:
ok, I'll will undo and disable the warning